dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/formModel.vue

```diff
@@ -550,7 +554,18 @@
         backfillRefresh: true,
         // whether this is a new Task
         isNewCreate: true,
-        tasksTypeList: Object.keys(tasksType)
+        tasksTypeList: Object.keys(tasksType),
+        helpUrlEnable: function (typeKey) {
+          let type = tasksType[typeKey]
+          if (type) {
+            let disabled = !!type.helperLinkDisable
+            return !disabled
+          }
+          return false
```

Comment:
if (!type) return false

const disabled = !! type.helperLinkDisable
return !disabled

There are three inversion operations here, can't it be simplified?

Reply:
ok , has been modified
``` javascript
helpUrlEnable: function (typeKey) {
          const type = tasksType[typeKey]
          if (type) {
            if (!type.helperLinkDisable) return true
            return !type.helperLinkDisable
          }
          return false
        },
```